_drafts/community.md

```diff
@@ -0,0 +1,73 @@
+Everyone seemed to be in agreement that (1) the community is one of R's biggest strengths and (2) a lot within the R community happens on twitter. During discussion, [Julia Lowndes](https://twitter.com/juliesquid) mentioned she joined twitter because she heard that people asked and answered questions about R there, and others echoed this sentiment. Simply, the R community is _not_ just for 'power users' or developers. It's a place for users and people interested in learning more about R. So, if you want to get involved in the community and you are not already, consider [getting a twitter account](https://support.twitter.com/articles/100990) and check out the [#rstats hashtag](https://twitter.com/search?q=%23rstats). We expect you'll be surprised by how responsive, welcoming, and inclusive the community is. 
+
+In addition to twitter, there are _many_ resources available within the R community where you can learn more about all things R. Below is a _brief_ list of resources mentioned during our discussion that had helped us feel more included in the community. Feel free to suggest more! 
+
+
+- **[R-Ladies](https://rladies.org/)** - - a world-wide organization focused on promoting gender diversity within the R community, with more than 30 local chapters
+- **Local R meetup groups** - a google search may show that there's one in your area! If not, maybe consider starting one! Face-to-face meet-ups for users of all levels are incredibly valuable
+- **[Rweekly](https://rweekly.org/)** - an incredible weekly recap of all things R
+- **[R-bloggers](https://www.r-bloggers.com/)** - an awesome resource to find posts from many different bloggers using R
+- **[DataCarpentry](http://www.datacarpentry.org/)** and **[Software Carpentry](https://software-carpentry.org/)**  - a resource of openly available lessons that promote and model reproducible research 
+- **[Stack Overflow](https://stackoverflow.com/)** - chances are your R question has already been answered here (with additional resources for people looking for jobs) 
+
+## Improving inclusivity
+
+No community is perfect, and being willing to consider our shortcomings and think about ways in which we can improve is so important. The group came up with a lot of great suggestions, including many I had not previously thought of personally.
+
+[Alice Daish](https://twitter.com/alice_data) did a great job capturing the conversation and allowing for more discussion online:
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">Join the lunchtime <a href="https://twitter.com/hashtag/runconf17?src=hash">#runconf17</a> discussion about the <a href="https://twitter.com/hashtag/rstats?src=hash">#rstats</a> communities - what do we need to do to improve? <a href="https://t.co/ztbXxNfqU7">pic.twitter.com/ztbXxNfqU7</a></p>&mdash; Alice Data (@alice_data) <a href="https://twitter.com/alice_data/status/868198529134510080">May 26, 2017</a></blockquote>
+
+To summarize here:
+
```

Comment:
added -- I think the hashtag idea and including it in this post are both great ideas!